concent-vm/Vagrantfile

```diff
@@ -65,7 +64,7 @@ Vagrant.configure("2") do |config|
             if [ ! -d "$directory" ]; then
                 mkdir $directory
             fi
-            mountpoint -q $directory || sudo mount ${partition_name[$index]} $directory
+            mountpoint -q $directory || sudo mount ${partition_name[$index]} $directory && sudo chown -R vagrant:vagrant $directory
```

Comment:
Shouldn't this change really be in the pull request that adds golem?

Reply:
I'm still not sure but I'm keeping it - it seems harmless and I don't want to have to retest everything without it.